js/colourpicker.js

```diff
@@ -369,129 +386,209 @@
 
       }
 
-      var hue, saturation, brightness, x, y, r, phi,
-
-      hex = input.val(),
+      var hue, saturation, brightness, x, y, r, phi, rgb, hsb, alpha, hex,
 
       // Helpful references
-      colourpicker = input.parent(),
+      colourpicker = input.parent();
```

Comment:
I was very confused by the long list of variables at some point and made a bunch of commas into semi-colons, but then I figured out what was going on and changed them back. I'll fix this.